my-thai-star/index.asciidoc

```diff
@@ -0,0 +1,23 @@
+= My Thai Star

```

Comment:
You can now add a subtitle with the == syntax. See https://github.com/devonfw-tutorials/tutorials/wiki/Development